pcdsdevices/areadetector/detectors.py

```diff
@@ -82,14 +126,12 @@ class PCDSAreaDetector(PCDSAreaDetectorBase):
     detector.
     """
 
-    image1 = Cpt(ImagePlugin, 'IMAGE1:', read_attrs=['array_data'],
```

Comment:
`read_attrs` no longer needs to be specified for these?

Reply:
`read_attrs` is no longer the preferred way to specify component kinds, it is better to just use the kind argument.

But actually in this particular instance, this specification was completely incorrect... the full image should never enter the bluesky data stream.

Reply:
I want to revisit the kinds here after the ophyd overlay class is done. I started a branch for it.